src/Components/Facility/ConsultationForm.tsx

```diff
@@ -364,7 +364,7 @@ export const ConsultationForm = (props: any) => {
 
   return (
     <div className="px-2 pb-2 max-w-3xl mx-auto">
-      <PageTitle title={headerText} hideBack={true} />
```

Comment:
consultation form, even though this is a page you have to click buttons to reach, did not have a back button. So adding it.